bin/fuel-convert

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import argparse
+import importlib
+import pkgutil
+
+from fuel import converters
+
+
+if __name__ == "__main__":
+    dataset_options = [
+        m[1] for m in pkgutil.iter_modules(converters.__path__) if not m[2]]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("dataset", help="what dataset to convert",
+                        choices=dataset_options)
+    parser.add_argument("-d", "--directory", help="directory in which input " +
+                        "files reside", type=str, default=None)
+    parser.add_argument("-o", "--output-file", help="where to save the " +
+                        "dataset", type=str, default=None)
+    args = parser.parse_args()
+
+    converter_module = importlib.import_module(
+        'fuel.converters.{}'.format(args.dataset))
```

Comment:
Argh. This DDoS attack on GitHub is driving me crazy.

Anyway, how about we just name the functions after the dataset they convert, e.g. `mnist(directory, save_path)`, and then put those functions in an `__all__` list. That way there's no scraping whatsoever required.